lib/src/util/react_wrappers.dart

```diff
@@ -273,9 +273,10 @@ ReactElement cloneElement(ReactElement element, [Map props, Iterable children])
   }
 }
 
-/// Returns the native Dart [ReactDartComponentInternal.component] associated with a [ReactComponent]
-/// [instance], or `null` if the component is not Dart-based _(an [Element] or a JS composite component)_.
-react.Component getDartComponent(/* [1] */ instance) {
+/// Returns the native Dart [ReactComponent.dartComponent] if [instance] is a [ReactComponent].
+///
+/// Returns `null` if the [instance] is not Dart-based _(an [Element] or a JS composite component)_.
+react.Component getDartComponent(/* ReactElement|ReactComponent */ instance) {
```

Comment:
This is actually `ReactElement|ReactComponent|Element`, since we support and anticipate DOM component instances being passed in here.